Dockerfile

```diff
@@ -1 +1,30 @@
+# Author Ashwin Babu 12/8/22
 FROM jupyter/scipy-notebook
+
+RUN pip install docopt-ng==0.8.1 \
+    && pip install vl-convert-python==0.5.0
+
+RUN conda install python-graphviz -y \
+    && conda install requests[version='>=2.24.0'] -y \
+    && conda install scikit-learn -y \
+    && conda install selenium[version='<4.3.0'] -y \
+    && conda install lightgbm -y \
+    && conda install pip -y \
+    && conda install jinja2 -y \
+    && conda install ipykernel -y \
+    && conda install jsonschema=4.16 -y \
+    && conda install -c conda-forge altair_saver -y \
+    && conda install pandas[version='<1.5'] -y \
+    && conda install matplotlib[version='>=3.2.2'] -y \
+    && conda install graphviz -y \
+    && conda install -c anaconda docopt -y \
```

Comment:
Not sure if we need to install `docopt` here since you already installed `docopt-ng` above.

Reply:
Edited.